sdk/keyvault/azkeys/client.go

```diff
@@ -153,19 +164,36 @@ type CreateECKeyOptions struct {
 	// Elliptic curve name. For valid values, see PossibleCurveNameValues.
 	CurveName *CurveName `json:"crv,omitempty"`
 
-	// Application specific metadata in the form of key-value pairs.
+	// Tags contain application specific metadata in the form of key-value pairs.
 	Tags map[string]string `json:"tags,omitempty"`
 
 	// Whether to create an EC key with HSM protection
 	HardwareProtected *bool
+
+	// The attributes of a key managed by the key vault service.
+	Properties *Properties  `json:"attributes,omitempty"`
+	Operations []*Operation `json:"key_ops,omitempty"`
+
+	// The policy rules under which the key can be exported.
+	ReleasePolicy *ReleasePolicy `json:"release_policy,omitempty"`
 }
 
 // convert CreateECKeyOptions to generated.KeyCreateParameters
 func (c *CreateECKeyOptions) toKeyCreateParameters(keyType KeyType) generated.KeyCreateParameters {
+	var keyOps []*generated.JSONWebKeyOperation
+	if c.Operations != nil {
+		keyOps = make([]*generated.JSONWebKeyOperation, len(c.Operations))
```

Comment:
For future you can just loop and append() - you don't even need to initialize 'keyOps' since append works with a nil slice ptr just fine.

Reply:
(you might have just been trying to be more efficient, however)

Reply:
This was an efficiency thing. I didn't want to create a slice that would be nil, and if it's left as nil the value won't be serialized when we write the request body